docs/README.md

```diff
@@ -16,6 +16,7 @@
 | [Conformance](conformance.md)                      | OCP/Kubernetes e2e tests                  | None                                  |
 | [Namespaces per cluster](namespaces-per-cluster.md) | Maximum Namespaces                       | None                                  |
 | [Services per namespace](services-per-namespace.md) | Maximum services per namespace           | None                                  |
+| [FIO I/O test](fio.md)                              | FIO I/O test - stress storage backend    | Privileged Containers, Working storage class |
```

Comment:
We need to add the workload to the CI Pass/Fail section as well.

Reply:
Ok, done